react-main/src/scss/equalizer.scss

```diff
@@ -0,0 +1,52 @@
+$equalizer-black: #000;
+$equalizer-grey: #999;
+
+* {
+    box-sizing: border-box;
+}
+
+body {
```

Comment:
Avoid defining styles for tags. Use classes instead (apply this comment to the whole file)